src/StoreKeeper/WooCommerce/B2C/Imports/AbstractProductImport.php

```diff
@@ -245,7 +245,7 @@ protected function cleanDotPath(string $shop_product_path): string
     /**
      * @throws ProductImportException
      */
-    protected function processItem($dotObject, array $options = [])
+    protected function processItem($dotObject, array $options = []): ?int
```

Comment:
it retrurn the wrong id 

```
$woocommerceProductId;
```
should be `$shopProductId`

Reply:
Not used anywhere anyway but changed it to $shopProductId.